geonode/settings.py

```diff
@@ -881,7 +881,6 @@
     'Tiles',
     'GML',
     'GZIP',
-    'ASCII',
```

Comment:
It was my code, I think it should be fine, and it was not specific to qgis server backend also